src/context/AuthContext.js

```diff
@@ -9,6 +9,7 @@ const AuthContext = createContext();
 export default AuthContext;
 export const AuthProvider = ({ children }) => {
   const [loading, setLoading] = useState(true);
+  const [status, setStatus] = useState("typing");
```

Comment:
This is good work. Same logic would also be required when submitting Interview Experiences.